functions-and-operators/locking-functions.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Locking Functions
+summary: Learn about user-level locking functions in TiDB.
+---
+
+# Locking Functions
+
+TiDB supports most of the user-level [locking functions](https://dev.mysql.com/doc/refman/5.7/en/locking-functions.html) available in MySQL 5.7.
+
+## Supported functions
+
+| Name                                                                                                                 | Description                                                           |
+|:---------------------------------------------------------------------------------------------------------------------|:----------------------------------------------------------------------|
+| [`GET_LOCK(lockName, timeout)`](https://dev.mysql.com/doc/refman/5.7/en/locking-functions.html#function_get-lock)    | Acquire an advisory lock. LockName must be <= 64 characters.          |
+| [`RELEASE_LOCK(lockName)`](https://dev.mysql.com/doc/refman/5.7/en/locking-functions.html#function_release-lock)     | Release a previously acquired lock LockName must be <= 64 characters. |
+| [`RELEASE_ALL_LOCKS()`](https://dev.mysql.com/doc/refman/5.7/en/locking-functions.html#function_release-all-locks)   | Release all locks held by the current session.                        |
+
+## MySQL compatibility
+
+* The minimum timeout permitted by TiDB is 1 second, and the maximum timeout is 1 hour. This differs from MySQL, where both zero second and unlimited timeouts are permitted. TiDB will automatically convert out of range values to the nearest permitted value.
+* TiDB does not automatically detect deadlocks in user-level locks. Deadlocked sessions will timeout after a maximum of 1 hour, but can also be manually resolved by using `KILL` on one of the affected sessions. Deadlocks can also be prevented by always acquiring user-locks in the same order.
```

Comment:
I am confused about "by using `KILL` on one of the affected sessions". Does it mean users need to use `KILL` only on one affected session out of multiple affected sessions? 
```suggestion
* TiDB does not automatically detect deadlocks caused by user-level locks. Deadlocked sessions will timeout after a maximum of 1 hour, but can also be manually resolved by using `KILL` on one of the affected sessions. You can also prevent deadlocks by constantly acquiring user-level locks in the same order.
```

Reply:
changed back to "always"